CHANGELOG.md

```diff
@@ -0,0 +1,182 @@
+# Changelog
+
+This changelog consolidates changes for the [nilrt](https://github.com/ni/nilrt) repo, [meta-nilrt](https://github.com/ni/meta-nilrt) OE layer, and NI's branch of the [linux](https://github.com/ni/linux) kernel. Other OE layer changes are tracked in their respective changelogs.
+
+The NI Linux Real-Time project uses a mainline branching model with product branches for stable releases. The distro itself is versioned according to a yearly release cadence and *is not* semantically versioned.
```

Comment:
I might suggest just removing the sentence on yearly release, since we may decide to change it more or less often, include bug fixes/patches, etc.

Reply:
Fixed.